src/app/core/entity-details/related-entities/related-entities.component.ts

```diff
@@ -112,12 +112,13 @@ export class RelatedEntitiesComponent<E extends Entity> implements OnInit {
     this.data = await this.getData();
     this.filter = this.initFilter();
 
-    if (this.showInactive === undefined) {
-      // show all related docs when visiting an archived entity
-      this.showInactive = this.entity.anonymized;
-    }
-
     this.listenToEntityUpdates();
+
+    //need to pass relevantValue for ActivitiesOverview
+    const titleColumn = this._columns?.find((col) => col.id === "title");
+    if (titleColumn?.additional?.relevantValue) {
+      titleColumn.additional.relevantValue = this.entity?.getId();
+    }
```

Comment:
Not sure if the required details are available there, but did you see if we can add this init code to the `EditTextWithAutocompleteComponent` instead of here?

Reply:
let me check